src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const key in actions) {
+    switch (actions[key].type) {
+      case 'addProperties':
+        const extraData = Object.assign({}, actions[key].extraData);
+
+        Object.assign(state, extraData);
```

Comment:
```suggestion
        Object.assign(state, action.extraData);
```

Reply:
also in this case code can pass the test only with my variant